runtime/jcl/common/shared.c

```diff
@@ -1890,8 +1890,54 @@ Java_com_ibm_oti_shared_SharedClassStatistics_freeSpaceBytesImpl(JNIEnv* env, jo
 #endif
 }
 
+jstring JNICALL
+Java_com_ibm_oti_shared_SharedClassStatistics_cachePathImpl(JNIEnv *env, jobject thisObj)
+{
+#if defined(J9VM_OPT_SHARED_CLASSES)
+	J9JavaVM *vm = ((J9VMThread *)env)->javaVM;
+	J9SharedClassJavacoreDataDescriptor descriptor;
+	jstring result = NULL;
+
+	Trc_JCL_com_ibm_oti_shared_SharedClassStatistics_cachePathImpl_Entry(env);
+
+	if (vm->sharedClassConfig) {
```

Comment:
Please make the NULL test explicit: `NULL != vm->sharedClassConfig`.
Also on line 1925 below.